GitHubService/GitHubService.API/Services/OrganizationService/IOrganizationService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using GitHubService.Domain.AggregatesModel.OrganizationAggregate;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GitHubService.API.Services.OrganizationService
+{
+    public interface IOrganizationService
+    {
+        public Organization GetOrganizationInformation(string name);
```

Comment:
C# 에서 interface 는 public 메소드 만을 알려주기 때문에 public 을  사용하지 않습니다!

```suggestion
        Organization GetOrganizationInformation(string name);
```